discord_thread_summarizer_using_openai/main.bal

```diff
@@ -0,0 +1,69 @@
+import ballerina/io;
+import ballerina/log;
+import ballerina/time;
+
+const TYPE_FORUM_CHANEL = 15;
+const time:Seconds ONE_DAY_IN_SECONDS = 86400;
+
+configurable int forumChannelId = ?;
+
+function readThreads(ActiveThreads activeThreads) returns error? {
+
+    foreach ChannelThread thread in activeThreads.threads {
+
+        if !check hasRecentMessages(thread) {
+            continue;
+        }
+
+        string prompt = check constructPrompt(thread);
+
+        string|error generateChatCompletionResult = generateChatCompletion(prompt);
+
+        if generateChatCompletionResult is string {
+            io:println(generateChatCompletionResult);
+        } else {
+            log:printError("Error occured when connecting to OpenAI: ",
+                generateChatCompletionResult,
+                stackTrace = generateChatCompletionResult.stackTrace()
```

Comment:
```suggestion
                generateChatCompletionResult.stackTrace()
```

May be able to merge this with the line above too.

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/127569dddc0e08bfed7f9a3baf48af8edd4719f2